accounting/overview/process_overview/supplier_bill.rst

```diff
@@ -1,9 +1,144 @@
+    amount in the bottom left table to adjust and match.
+
+Validate The Vendor Bill
+------------------------
+
+Once the vendor bill is validated, a journal entry will be generated
+based on the configuration on the invoice. This journal entry may differ
+depending on the the accounting package you choose to use.
+
+For most European countries, the journal entry will use the following
+accounts:
+
+-  **Accounts Payable:** defined on the vendor form
+
+-  **Taxes:** defined on the products and per line
+
+-  **Expenses:** defined on the line item product used
+
+For Anglo-Saxon (US) accounting, the journal entry will use the
+following accounts:
+
+-  **Accounts Payable:** defined on the vendor form
+
+-  **Taxes:** defined on the products and per line
+
+-  **Goods Received:** defined on the product form
+
+You can check your Profit & Loss or the Balance Sheet reports after
+having validated a couple of vendor bills to see the impact on your
+general ledger.
+
+Pay a bill
+----------
+
+To create a payment for an open vendor bill directly, you can Register a
+Payment at the top of the form.
+
+From there, you select the payment method (i.e. Checking account, credit
+card, check, etc…) and the amount you wish to pay. By default, Odoo will
+propose the entire remaining balance on the bill for payment. In the
+memo field, we recommend you set the vendor invoice number as a
+reference(Odoo will auto fill this field from the from the vendor bill
+if set it correctly).
+
+.. image:: ./media/vendor_bill03.png
+   :align: center
+
+
+.. note::
+    You can also register a payment to a vendor directly without applying it to a vendor bill. 
+    To do that, :menuselection:`Purchases --> Payments`. Then, 
+    from the vendor bill you will be able to reconcile this payment with directly.
+
+Printing vendor Checks
+----------------------
+
+If you choose to pay your vendor bills by check, Odoo offers a method to
+do so directly from your vendor payments within Odoo. Whether you do so
+on a daily basis or prefer to do so at the end of the week, you can
+print in checks in batches.
+
+If you have checks to print, Odoo’s accounting dashboard acts as a to do
+list and reminds you of how many checks you have left to be printed.
+
+.. image:: ./media/vendor_bill04.png
+   :align: center
+
+By selecting the amount of checks to be printed, you can dive right into
+a list of all payments that are ready to be processed.
+
+Select all the checks you wish to print (use the first checkbox to
+select them all) and set the action to **Print Checks**. Odoo will ask you
+to set the next check number in the sequence and will then print all the
+checks at once.
+
+.. image:: ./media/vendor_bill05.png
+   :align: center
+
+Reporting
+=========
+
+Aged payable balance
+--------------------
+
+In order to get a list of open vendor bills and their related due dates,
+you can use the Aged Payable report, under the reporting menu, (in
```

Comment:
"Aged Payable" in bold (usually when using CamelCase, is either a button or a field :wink: )